### DISCUSSION TITLE
What is the method used for conditional field generation?

### DISCUSSION THREAD

**Discussion by flydream0428** [GeoStat-Framework/GSTools#304] [Q&A] [answered]
Is it Sequential Gaussian Simulation?

**Comment by MuellerSeb** [comment, marked as answer]
Hey there,

no, it is not sequential gaussian. We use the randomization method the generate a random field with a variance of `1` and zero mean. Then we use kriging to calculate the kriging field and the error variance and then we multiply the error-variance with the random field and add it to the kriging field. This is also described in a [separate section](https://gmd.copernicus.org/articles/15/3161/2022/#Ch1.S2.SS2.SSS3) in the paper.

This is much more efficient than sequential gaussian because it is not depending on the point order for execution.

Do you need further details?

Cheers,
Sebastian

**Reply by flydream0428** [reply]
Thanks a lot, Sebastian. I forgot to check the paper:)
